pkgs/top-level/all-packages.nix

```diff
@@ -1022,9 +1022,7 @@ let
 
   checkinstall = callPackage ../tools/package-management/checkinstall { };
 
-  cheetahTemplate = builderDefsPackage (import ../tools/text/cheetah-template/2.0.1.nix) {
-    inherit makeWrapper python;
-  };
+  cheetahTemplate = pythonPackages.cheetah;
```

Comment:
We have a place for those kind of aliases at the very end of the file. Would you mind moving this definition there (with a time stamp)?